State_manager_code/StateManager.py

```diff
@@ -93,9 +105,9 @@ def __init__(self):
         self.current_state = States.Start
         self.StateStartTime = time.time()
 
-    def SetTimeOut(self, decision_time, return_time):
-        self.TransitionTimeOut[States.TrialStarted] = decision_time
-        self.TransitionTimeOut[States.WaitForReturn] = return_time
+    def SetTimeOuts(self, decision_time, return_time):
+        self.decision_time = decision_time
```

Comment:
This is an ok change too :-), please do it again on top of the latest StateManager version from main

Reply:
ok